### ISSUE TITLE
NVDA does not announce slider values in Google Chrome while in Focus mode

### ISSUE THREAD

**Issue by pitermach** [nvaccess/nvda#6323]
Chrome Canary 55, NVDA next-13518,19c296f3 on Windows 10 X64

When going into focus mode to adjust a Slider,  NVDA does not announce its value either on it gaining focus or getting changed with the arrow keys. I'd have put this to Chrome being Chrome, but Narrator, which doesn't officially support it does read the slider value and NVDA is also seeing it according to NVDA+F1. Before submitting this ticket I tested 2 sites. On Youtube and on http://websdr.ewi.utwente.nl:8901. On Youtube, NVDA does not indicate the value of the seek slider in either browse or Focus Mode, while on the web SDR it reads the value of the volume slider only while in browse mode. Following is the information from NVDA+F1 for both cases.

INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (16:04:15):
Developer info for navigator object:
name: u'Seek slider'
role: ROLE_SLIDER
states: STATE_FOCUSABLE, STATE_EDITABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorIa2WebIAccessible object at 0x0588E570>
Python class mro: (<class 'NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorIa2WebIAccessible'>, <class 'NVDAObjects.behaviors.EditableTextWithAutoSelectDetection'>, <class 'NVDAObjects.behaviors.EditableText'>, <class 'editableText.EditableText'>, <class 'NVDAObjects.IAccessible.ia2Web.Editor'>, <class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: (40, 484, 616, 5)
value: u'0:06 of 18:11'
appModule: <'appModuleHandler' (appName u'chrome', process ID 3556) at address 58ceb50>
appModule.productName: u'Google Chrome'
appModule.productVersion: u'55.0.2845.0'
TextInfo: <class 'NVDAObjects.IAccessible.ia2TextMozilla.MozillaCompoundTextInfo'>
windowHandle: 5179296L
windowClassName: u'Chrome_RenderWidgetHostHWND'
windowControlID: -851871632
windowStyle: 1445986304
windowThreadID: 8848
windowText: u'Chrome Legacy Window'
displayText: u''
IAccessibleObject: <POINTER(IAccessible2) ptr=0x81f8fe4 at 58966c0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=5179296, objectID=-4, childID=-44213
IAccessible accName: u'Seek slider'
IAccessible accRole: ROLE_SYSTEM_SLIDER
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: u'0:06 of 18:11'
IAccessible2 windowHandle: 5179296
IAccessible2 uniqueID: -44213
IAccessible2 role: ROLE_SYSTEM_SLIDER
IAccessible2 states: IA2_STATE_OPAQUE, IA2_STATE_HORIZONTAL, IA2_STATE_EDITABLE (1048)
IAccessible2 attributes: u'display:block;tag:div;xml-roles:slider;explicit-name:true;valuetext:0\:06 of 18\:11;class:ytp-progress-bar;'

INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (16:16:02):
Developer info for navigator object:
name: None
role: ROLE_SLIDER
states: STATE_FOCUSABLE, STATE_EDITABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorIa2WebIAccessible object at 0x058CE650>
Python class mro: (<class 'NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorIa2WebIAccessible'>, <class 'NVDAObjects.behaviors.EditableTextWithAutoSelectDetection'>, <class 'NVDAObjects.behaviors.EditableText'>, <class 'editableText.EditableText'>, <class 'NVDAObjects.IAccessible.ia2Web.Editor'>, <class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: (105, 300, 130, 22)
value: u'0'
appModule: <'appModuleHandler' (appName u'chrome', process ID 3556) at address 58ceb50>
appModule.productName: u'Google Chrome'
appModule.productVersion: u'55.0.2845.0'
TextInfo: <class 'NVDAObjects.IAccessible.ia2TextMozilla.MozillaCompoundTextInfo'>
windowHandle: 919546L
windowClassName: u'Chrome_RenderWidgetHostHWND'
windowControlID: -708004704
windowStyle: 1445986304
windowThreadID: 8848
windowText: u'Chrome Legacy Window'
displayText: u''
IAccessibleObject: <POINTER(IAccessible2) ptr=0x8221b2c at 58bdf30>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=919546, objectID=-4, childID=-123065
IAccessible accName: None
IAccessible accRole: ROLE_SYSTEM_SLIDER
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: u'0'
IAccessible2 windowHandle: 919546
IAccessible2 uniqueID: -123065
IAccessible2 role: ROLE_SYSTEM_SLIDER
IAccessible2 states: IA2_STATE_OPAQUE, IA2_STATE_HORIZONTAL, IA2_STATE_EDITABLE (1048)
IAccessible2 attributes: u'display:inline-block;tag:input;xml-roles:slider;valuetext:0;id:volumecontrol2;'

**Comment by wbreuschel** [comment]
This is happening with properly marked up ARIA sliders. Example: http://www.oaa-accessibility.org/examplep/slider1/

It's also happening with HTML5 range inputs. Example: https://patrickhlauke.github.io/aria/demos/html5-slider.html

Chrome 57.0.2987.133
NVDA 2017.1

**Comment by jcsteh** [comment]
Confirmed. Chrome is incorrectly exposing the editable state for sliders.

Minimal test case: `data:text/html,<div tabindex="0" role="slider" aria-valuenow="100" style="height: 5px;"></div>`

**Comment by jcsteh** [comment]
Fixed in Chrome Canary: Version 61.0.3122.0 (Official Build) canary (64-bit). So, this should be in Chrome 61 when it's released.
